src/shared/components/VideoPreviewBase/VideoPreviewBase.tsx

```diff
@@ -345,6 +349,11 @@ const VideoPreviewBase: React.FC<VideoPreviewBaseProps> = ({
                   <SvgGlyphMore />
                 </KebabMenuIconContainer>
                 <ContextMenu contextMenuOpts={contextMenuOpts}>
+                  {onOpenInTabClick && (
+                    <ContextMenuItem icon={<SvgGlyphNewTab />} onClick={onOpenInTabClick}>
+                      Open in tab
+                    </ContextMenuItem>
```

Comment:
This still displays for drafts, we should display that only for published videos:
<img width="589" alt="Screenshot 2021-06-10 at 10 08 57" src="https://user-images.githubusercontent.com/12646744/121488972-e27c2000-c9d3-11eb-9450-47ee8d6a1b3d.png">